src/project.js

```diff
@@ -3,8 +3,7 @@ const alfy = require("alfy");
 const projectPaths = require("./project/paths");
 const projectName = require("./project/name");
 
-const buildItem = (product, projectPath) => {
-  const name = projectName.get(projectPath);
+const buildItem = (product, projectPath, name = projectName.get(projectPath)) => {
```

Comment:
Sorry I didn't understand the reason of this change. Can you explain it?

Reply:
this was my bad, let me revert the changes, because I was utilising `buildItem` to construct for `fileActions.js` output.